client/src/routes/Temple/components/VideoBase/VideoBase.tsx

```diff
@@ -5,6 +5,8 @@ import {Platform} from 'react-native';
 
 const StyledVideo = styled(RNVideo)({
   flex: 1,
+  shadowOpacity: 1,
+  shadowColor: 'transparent',
```

Comment:
Nice find! How about a comment here referring to the GitHub issue?

Reply:
Ah super, will fix!